docs/source/modules/mixedlayer.rst

```diff
@@ -0,0 +1,277 @@
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``do_ape_sst``                 | Flag to set prescribed SST according to the APE aquaplanet analytic form                                    | ``False``            |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``add_latent_heat_flux_anom``  | Flag to add an anomalous latent heat flux                                                                   | ``False``            |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+
+Diagnostics
+-------------------
++-------------------+-------------------------------------+------------------------------------+
+| Name              | Description                         | Units                              |
++===================+=====================================+====================================+
+| t_surf            | Surface temperature                 | :math:`\text{K}\`                  |
++-------------------+-------------------------------------+------------------------------------+
+| delta_t_surf      | Surface temperature change          | :math:`\text{K}\`                  |
++-------------------+-------------------------------------+------------------------------------+
+| flux_t            | Surface sensible heat flux          | :math:`\text{W}\,\text{m}^{-2}`    |
++-------------------+-------------------------------------+------------------------------------+
+| flux_lhe          | Surface latent heat flux            | :math:`\text{W}\,\text{m}^{-2}`    |
++-------------------+-------------------------------------+------------------------------------+
+| flux_oceanq       | Ocean heat flux                     | :math:`\text{W}\,\text{m}^{-2}`    |
++-------------------+-------------------------------------+------------------------------------+
+| ice_conc          | Sea ice concentration               | None                               |
++-------------------+-------------------------------------+------------------------------------+
+| albedo            | Surface albedo                      | None                               |
++-------------------+-------------------------------------+------------------------------------+
+
+
+Relevant modules and subroutines
+--------------------------------
+.. List the names of relevant modules, subroutines, functions, etc.
+.. You can add also code snippets, using Sphinx code formatting
+
+The mixed layer code is located in: src/atmos_spectral/driver/solo/mixed_layer.F90. The name of this file reflects the fact that the code determines the properties of the single layer (either a slab ocean model 
```

Comment:
The file locations here (4 total) don't jump out at me. Suggest using ``link`` notation to make them red.